src/frontend/sharedComponents/BottomSheetModal/index.tsx

```diff
@@ -30,6 +30,9 @@ export const useBottomSheetModal = ({openOnMount}: {openOnMount: boolean}) => {
   const openSheet = React.useCallback(() => {
     if (sheetRef.current) {
       setIsOpen(true);
+      if (Keyboard.isVisible()) {
+        Keyboard.dismiss();
+      }
```

Comment:
@achou11 the keyboard was persisting when the modal opened. I added this so that it consistently gets dismissed before opening the modal